typings/index.d.ts

```diff
@@ -805,6 +805,7 @@ declare module 'discord.js' {
 		public readonly deletable: boolean;
 		public guild: Guild;
 		public readonly manageable: boolean;
+		public readonly members: Collection<Snowflake, GuildMember>
```

Comment:
nit: Missing semicolon 😅 
```suggestion
		public readonly members: Collection<Snowflake, GuildMember>;
```

Reply:
Appreciate your contribution!
Although it looks like the PR is not going to be merged cause they are going to refactor their method of defining the property.
Most likely removing it from GuildChannel and adding it to TextChannel.